docs/iot/TOC.md

```diff
@@ -0,0 +1,34 @@
+# [IoT and Tizen IoT](/iot/overview.md)
```

Comment:
I can't find this file. Do you mean `/iot/index.md`?

Reply:
done